org.freedesktop.Sdk.Extension.dotnet.yaml

```diff
@@ -0,0 +1,76 @@
+app-id: org.freedesktop.Sdk.Extension.dotnet
+branch: '18.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+separate-locales: false
+appstream-compose: false
+modules:
+  - name: libunwind
+    buildsystem: autotools
+    build-options:
+      prefix: /usr/lib/sdk/dotnet
+    cleanup:
+      - '/include'
+      - '/pkgconfig'
+      - '*.a'
+      - '*.la'
+      - '*ptrace.so*'
+      - '*setjmp.so*'
+    sources:
+      - type: archive
+        url: https://download.savannah.nongnu.org/releases/libunwind/libunwind-1.2.tar.gz
+        sha256: 1de38ffbdc88bd694d10081865871cd2bfbb02ad8ef9e1606aee18d65532b992
+  - name: dotnet-sdk
+    buildsystem: simple
+    build-commands:
+      - 'mkdir -p /usr/lib/sdk/dotnet/{bin,lib}'
+      - 'cp -r * /usr/lib/sdk/dotnet/lib'
+      - 'ln -s /usr/lib/sdk/dotnet/lib/dotnet /usr/lib/sdk/dotnet/bin/dotnet'
+    sources:
+      - type: archive
+        only-arches: [x86_64]
+        url: https://download.visualstudio.microsoft.com/download/pr/e85de743-f80b-481b-b10e-d2e37f05a7ce/0bf3ff93417e19ad8d6b2d3ded84d664/dotnet-sdk-2.1.403-linux-x64.tar.gz
+        sha512: 903a8a633aea9211ba36232a2decb3b34a59bb62bc145a0e7a90ca46dd37bb6c2da02bcbe2c50c17e08cdff8e48605c0f990786faf1f06be1ea4a4d373beb8a9
+      - type: archive
+        only-arches: [arm]
+        url: https://download.visualstudio.microsoft.com/download/pr/10b96626-02d8-415a-be85-051a2a48d0c2/5ec51d3d9f092ba558fb5f1f03d26699/dotnet-sdk-2.1.403-linux-arm.tar.gz
+        sha512: 144f1a8822f57d3e9286ed80b7ee0c7796cb2765632dd99d0f89a276248c3be77d8d90c8bf987af0237e67f2af26be834c138441980d2c7d35156ee1bb7e3d94\
+      - type: arches
+        only-arches: [aarch64]
+        url: https://download.visualstudio.microsoft.com/download/pr/00038a67-bb86-4c39-88df-7c0998002a9e/97de51fd691c68e18ddd3dcaf3d60181/dotnet-sdk-2.1.403-linux-arm64.tar.gz
+        sha512: a9eeffeca6b80b2873c21c7b8bb06df096290cd5b23a6e1e0fb158b60f2c2af6ee4b2bfcfd96a303ebf33ea2cc782f3dee629d06d48b634dda05ae9e11051335
+  - name: scripts
+    buildsystem: simple
+    build-commands:
+      - 'cp enable.sh install.sh install-sdk.sh /usr/lib/sdk/dotnet'
+    sources:
+      - type: script
+        commands:
+          - 'mkdir -p /app/dotnet/{bin,lib}'
+          - 'cp -L /usr/lib/sdk/dotnet/lib/libunwind{,-coredump,-x86_64}.so /app/dotnet/lib'
```

Comment:
Does `-x86_64` variant exist on all architectures? If not, does it mean the command will fail?